packages/mui-material/src/styles/createTheme.test.js

```diff
@@ -23,6 +56,60 @@ describe('createTheme', () => {
     expect(theme.palette.secondary.main).to.equal(green.A400);
   });
 
+  describe('CSS variables', () => {
+    it('should have a light as a default colorScheme if only `palette` is provided', () => {
```

Comment:
Ah nice, this means we can use the old theme structure and still have vars generated. This is great!

Reply:
Yes, that's one of the goal.